src/common/src/buffer/bitmap.rs

```diff
@@ -291,6 +291,13 @@ impl Bitmap {
         Bitmap::from_bytes_with_num_bits(bits, lhs.num_bits)
     }
 
+    pub fn ones(&self) -> impl Iterator<Item = usize> + '_ {
+        self.iter()
+            .enumerate()
+            .filter(|(_, bit)| *bit)
```

Comment:
what about using `filter_map` here?

Reply:
IMO `filter` and `map` is more concise here.